conf/timeseries-dev-source.conf

```diff
@@ -1,5 +1,27 @@
     dataset = "prometheus"
+
+    # Should not change once dataset has been set up on the server and data has been persisted to cassandra
+    definition {
```

Comment:
More comments should be added here.  That this is for defining the schema used for ingesting data and persistence.  Also it is not just when data has been persisted to Cassandra, also once data is written into Kafka.
That changing the definition after data is ingested will lead to corruption and inability to ingest or query data.

Also please add comment for what partition-columns and data-columns are.